common-rendering/src/components/keyEvents.tsx

```diff
@@ -109,44 +104,34 @@ const timeTextWrapperStyles: SerializedStyles = css`
 	margin-left: ${remSpace[4]};
 `;
 
-const keyEventsTextGrayBackground = (theme: ArticleTheme): string => {
-	switch (theme) {
-		case ArticlePillar.News:
-		case ArticlePillar.Lifestyle:
-		case ArticlePillar.Sport:
-			return getColor(theme, 400);
-		default:
-			return getColor(theme, 300);
-	}
-};
 
 const textStyles = (
-	theme: ArticleTheme,
+	format: ArticleFormat,
 	supportsDarkMode: boolean
 ): SerializedStyles => css`
 	${headline.xxxsmall({ fontWeight: "regular", lineHeight: "regular" })};
-	color: ${getColor(theme, 400)};
+	color: ${text.keyEventsWhiteBackground(format)};
 
 	text-decoration: none;
 
 	&:hover {
-		color: ${getColor(theme, 400)};
+		color: ${text.keyEventsWhiteBackground(format)};
 		text-decoration: underline;
 	}
 
 	${from.desktop} {
-		color: ${keyEventsTextGrayBackground(theme)};
+		color: ${text.keyEventsGrayBackground(format)};
 
 		&:hover {
-			color: ${keyEventsTextGrayBackground(theme)};
+			color: ${text.keyEventsGrayBackground(format)};
 			text-decoration: underline;
 		}
 	}
 
 	${darkModeCss(supportsDarkMode)`
-		color: ${getColor(theme, 500)};
+		color: ${getColor(format.theme, 500)};
```

Comment:
I believe this needs to be handled in the separate ticket for dark mode because it's not 500. See here: https://www.chromatic.com/test?appId=5dfcbf3012392c0020e7140b&id=61e032c4708f77003a8b3c7d